cli/jobs/dataprep/dask/copy_nyc_taxi.py

```diff
@@ -0,0 +1,52 @@
+import uuid, time, os, json
+import io, sys, urllib.request
+from tqdm import tqdm
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--months", type=int, default=12)
+args = parser.parse_args()
+months = args.months
+
+cwd = os.getcwd()
+
+data_dir = os.path.abspath(os.path.join(cwd, "data"))
+if not os.path.exists(data_dir):
+    os.makedirs(data_dir)
+
+taxidir = os.path.join(data_dir, "nyctaxi")
+if not os.path.exists(taxidir):
+    os.makedirs(taxidir)
+
+filenames = []
+local_paths = []
+for i in range(1, months + 1):
```

Comment:
can we change this out reading from s3fs (or preferably Azure Blob) instead of assuming filenames + http reads?

Reply:
I moved the data to azure now, so we can delete this script